src/Tizen.Wearable.CircularUI.Chart.Forms/LegendPosition.cs

```diff
@@ -0,0 +1,42 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Wearable.CircularUI.Chart.Forms
+{
+
+    /// <summary>
+    /// Enumeration for postion of legend of PieChartView and DonutChartView
+    /// </summary>
+    public enum LegendPosition
```

Comment:
I think, None is need that does not display legend

Reply:
If user want to display legend, he can only add DataItem instead PieDataItem. and LegendPosition mean position not visibility. so We need to add another property like as "LengendIsVisible"

Reply:
Yes, We need `LegendIsVisible`

Reply:
Done